source/fundamentals/indexes.txt

```diff
@@ -371,3 +384,97 @@ The following example creates a unique, descending index on the ``theaterId`` fi
 
 Refer to the :manual:`Unique Indexes page </core/index-unique>` in the MongoDB server manual for more information.
 .. driver-content-end
+
+Remove an Index
+---------------
+
+You can remove any unused index that isn't a :ref:`unique index
+<unique-indexes>`. 
+
+The following sections show the three ways to remove indexes:
+
+- Using an index specification document
+- Using an indexed name field
+- Removing all indexes
+
+Index Specification Document
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pass the index specification documenent to the ``dropIndex()`` method to
+remove an index from a collection. 
+
+The following snippet removes an ascending index on the ``title`` field
+in a collection:
+
+.. code-block:: java
+
+   collection.dropIndex(Indexes.ascending("title"));
+
+.. important::
+
+   This does not work for a text index. To remove a text index, see the
+   :ref:`Name Field <name_field>` section.
+
+.. _name_field:
+
+Name Field
+~~~~~~~~~~
+
+Pass the indexed ``name`` field to the ``dropIndex()`` method to remove
+an index from a collection. You must do this to remove a text index.
+
+Use the ``listIndexes()`` method to see the indexed ``name`` fields in
+your collection. 
+
+The following snippet retrieves all the indexes in a collection, which
+has a text index on the ``title`` field:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin listIndex
+   :end-before: end listIndex
+
+Your output should look something like the following:
+
+.. code-block:: json
+
+   { "v": 2, "key": {"_id": 1}, "name": "_id_" }
+   { "v": 2, "key": {"_fts": "text", "_ftsx": 1}, "name": "title_text", "weights": {"title": 1}, 
+   "default_language": "english", "language_override": "language", "textIndexVersion": 3 }
+
+You can see a text index with the ``name`` "title_text" in the preceding output.
+
+The following snippet removes the "title_text" index in the collection:
+
+.. code-block:: java
+
+   collection.dropIndex("title_text");
+
+.. note:: 
+
+   You cannot remove a single field from a compound text index. You must
+   drop the entire index and create a new one to update the indexed
+   fields 
+
+Removing All Indexes
+~~~~~~~~~~~~~~~~~~~~
+
+Starting with MongoDB 4.2, you can drop all non-``_id`` indexes in a
```

Comment:
I used this term because it was used in the manual in the first note on this page: https://docs.mongodb.com/manual/reference/method/db.collection.dropIndex/#mongodb-method-db.collection.dropIndex